OnePlus/OP10R/res/xml/power_profile.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on.display0">0.1</item>
+    <item name="screen.on.display0">0.1</item>
+    <item name="screen.full.display0">0.1</item>
+    <item name="bluetooth.active">0.1</item>
```

Comment:
If your screen consumed only 0.1mA, your smartphone would have 1 year of battery life screen on.
Is it the case?
I'm guessing not. That means this power profile is wrong.